src/WebJobs.Script/Diagnostics/OpenTelemetry/OpenTelemetryConfigurationExtensions.cs

```diff
@@ -119,7 +118,6 @@ private static IOpenTelemetryBuilder ConfigureTracing(this IOpenTelemetryBuilder
                 if (enableAzureMonitor)
                 {
                     builder.AddAzureMonitorTraceExporter(opt => ConfigureAzureMonitorOptions(opt, azMonConnectionString, credential));
-                    builder.AddLiveMetrics(opt => ConfigureAzureMonitorOptions(opt, azMonConnectionString, credential));
```

Comment:
how is live metrics added now?

Reply:
It has been removed from AzMonDistro and is now part of the exporter package, where it is enabled by default.

Reply:
I've removed any dependency on the distro, as Live Metric was the sole reason we were utilizing it.